apps/src/templates/sessions/LockoutPanel.jsx

```diff
@@ -0,0 +1,280 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import Button from '../Button';
+import i18n from '@cdo/locale';
+import {getStore} from '../../redux';
+import {isEmail} from '@cdo/apps/util/formatValidation';
+import cookies from 'js-cookie';
+import * as color from '../../util/color';
+import headerImage from './images/lockout_penguin.png';
+import headerThanksImage from './images/dancing_penguin.png';
+
+/**
+ * This panel represents the page that is displayed to accounts that are being
+ * locked from accessing certain parts of the site until they get parental
+ * permission. This panel gives a form to request (or see details about a
+ * pending request for) parental permission.
+ */
+export default function LockoutPanel(props) {
+  // Set the disabled state of the submit button based on the validity of the
+  // email in the field.
+  const [disabled, setDisabled] = useState(() => !isEmail(props.pendingEmail));
+
+  // When the email field is updated, also update the disability state of the
+  // submit button.
+  const onEmailUpdate = event => {
+    setDisabled(!isEmail(event.target.value));
+  };
+
+  // This will set the email to the current pending email and fire off the
+  // form as though they had typed in the same email again.
+  const resendPermissionEmail = event => {
+    // Don't submit the form... we will do that ourselves.
+    event.preventDefault();
+
+    const field = document.getElementById('parent-email');
+    field.value = props.pendingEmail;
+    this.submit();
+  };
+
+  // Get the current locale.
+  const locale = cookies.get('language_') || 'en-US';
+
+  // Whether or not we are rendering right-to-left.
+  const isRTL = getStore().getState()?.isRtl;
+
+  // How to format any localized dates on the page.
+  const dateOptions = {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  };
+
+  // We want to mark the email in the prompt in bold, so we split the
+  // localized string. It is a slight hack since we cannot just embed React
+  // component code into a string. Should work as though we could though.
+  const pendingPrompt = i18n.sessionLockoutPendingPrompt({
```

Comment:
I wonder if the `EnhancedSafeMarkdown` component would work? I think that is how we render markdown strings in other React components. I think your solution works as well though.